src/melee/moves.ts

```diff
@@ -152,16 +158,20 @@ const moves = {
   },
 };
 
-export function getMoveInfo(moveId) {
-  return moves[moveId];
+export function getMoveInfo(moveId: number): Move {
+  const m = moves[moveId];
+  if (!m) {
+    throw new Error(`Could not find move with id: ${moveId}`);
```

Comment:
I don't like this. Our move definitions are not complete, being able to return "unknown" for undefined moves is actually used and this change would be a breaking change for some programs.

Reply:
I've updated the `moves.ts` file with an `UnknownMove` constant.